Doc/whatsnew/3.11.rst

```diff
@@ -493,6 +495,222 @@ Optimizations
   (Contributed by Inada Naoki in :issue:`46845`.)
 
 
+Faster CPython
+==============
+
+CPython 3.11 is on average `1.22x faster <https://github.com/faster-cpython/ideas/blob/main/main-vs-310.rst>`_
+than CPython 3.10 when measured with the
+`pyperformance <https://github.com/python/pyperformance>`_ benchmark suite, on
+Ubuntu Linux. Depending on your workload, the speedup could be 10-60% faster.
+
+The project focuses on two major areas in Python: faster startup and faster
+runtime. Other optimizations not under this project are listed in `Optimizations`_.
+
```

Comment:
Is it necessary to separate the Faster CPython section from the optimisations section? We could have it all under optimisations, with maybe a footnote thanking Microsoft for its sponsorship of some of the work.

Reply:
Good point - we definitely could. I only have cosmetic reasons for not doing that. Nested sections within sections really don't render well in our current scheme. See the [3.10 what's new for example](https://docs.python.org/3/whatsnew/3.10.html) .

![image](https://user-images.githubusercontent.com/28750310/161390784-5c900d7f-983a-42c9-b43d-5391a8759959.png)

_Logically_ it seems better to do what you said though.

Reply:
Wait sorry I think I misunderstood what you intended. If you meant that we could list them as bullet points under Optimizations, I'd prefer not to.  Some parts need more elaboration, and the bullet points don't help with logical separation as much as sections do.

Reply:
That brings me to my second question - is whatsnew the place to elaborate so much?  People read what's new to assess the impact of upgrading to this version.  The headline perf improvement numbers belong there, but I'm not sure about all the details. 

If we create a blog-type doc for Faster CPython (and link to it from whatsnew), then in future years we can add sections to it and it becomes a self-contained history of the project.

Reply:
Hmm, I think this definitely could be shorter. Where do you think I can cut some of cruft? Or maybe I can move this entire section off of whats new like you mentioned.

Frankly, I'm not sure of the policy regarding what's new. My current understanding is that whatever the release manager adds is what goes. For example [Python 3.8's whatsnew](https://docs.python.org/3/whatsnew/3.8.html) went into great deal of detail for things we might not consider today as _that_ newsworthy.

Reply:
The release manager is definitely the final editor of what's new.